amethyst_renderer/src/pass/util.rs

```diff
@@ -333,6 +335,36 @@ pub fn set_view_args(
     effect.update_constant_buffer("ViewArgs", &view_args.std140(), encoder);
 }
 
+/// Sets the view arguments in the constant buffer using the screen dimensions.
+pub fn set_view_args_screen(
+    effect: &mut Effect,
+    encoder: &mut Encoder,
+    screen_dimensions: &ScreenDimensions,
+) {
+    #[cfg(feature = "profiler")]
+    profile_scope!("render_setviewargsscreen");
+
+    let translation = Vector3::<f32>::new(0.0, 0.0, 1000.0);
```

Comment:
does 2000.0 work, and if it does, could you extract this and the one in the `Orthographic3` into a `pub const` (and export all the way up)?
That way people depending on magic numbers can at least have a constant to rely on